integration-tests/tests/run-notebook.js

```diff
@@ -41,18 +41,19 @@ const testRunNotebookFn = _.flow(
   // Therefore, we start with a 1000ms delay, then make sure there are no spinners just in case the
   // AJAX calls are unexpectedly slow.
   await delay(1000)
-  waitForNoSpinners(page)
+  await waitForNoSpinners(page)
   await click(page, clickable({ text: 'Create' }))
   await findElement(page, clickable({ textContains: 'Creating' }))
   await findElement(page, clickable({ textContains: 'Running' }), { timeout: 10 * 60 * 1000 })
 
   const frame = await findIframe(page)
   await findElement(frame, '//*[@title="Kernel Idle"]')
-  await fillIn(frame, '//textarea', 'print(123456789099876543210990+9876543219)')
+  await fillIn(frame, '//textarea', '100 + 1')
   await click(frame, clickable({ text: 'Run' }))
-  await findText(frame, '123456789099886419754209')
+  await findText(frame, '101')
```

Comment:
The original code looks for `123456789099886419754209` string in code output field to verify code executed successfully. But, the same string can be found in code input field as well. So we don't know what the `findText()` finds. 
Make code output uniquely different so that test finds and verify the right string.

Reply:
```
123456789099876543210990+9876543219
123456789099886419754209
             ^
   difference starts at this digit
```

Reply:
aha, I missed it. I will undo this change.